src/test/groovy/com/github/rodionovsasha/commentservice/services/UserServiceTest.groovy

```diff
@@ -0,0 +1,137 @@
+package com.github.rodionovsasha.commentservice.services
+
+import com.github.rodionovsasha.commentservice.BaseTest
+import com.github.rodionovsasha.commentservice.exceptions.InactiveUserException
+import com.github.rodionovsasha.commentservice.exceptions.UserNotFoundException
+import com.github.rodionovsasha.commentservice.repositories.UserRepository
+import org.springframework.beans.factory.annotation.Autowired
+
+class UserServiceTest extends BaseTest {
+    @Autowired
+    UserService userService
+    @Autowired
+    UserRepository userRepository
+
+    def "getActiveUser returns active user by id"() {
+        when:
+        def user = userService.getActiveUser(1)
+
+        then:
+        user.id == 1
+        user.name == "Homer"
+        user.age == 39
+        user.enabled
+    }
+
+    def "getActiveUser throws when user is inactive"() {
+        when:
+        userService.getActiveUser(2)
+
+        then:
+        def e = thrown(InactiveUserException)
+        e.message == "The user with id '2' is not active"
+    }
+
+    def "updateName does update user's name"() {
+        when:
```

Comment:
We don'r really see that name gets updated. I think we need a `given` section that shows it:
```groovy
given:
  userRepository.getOne(1L).name == "Homer"
```